Makefile

```diff
@@ -333,6 +341,7 @@ MATH_SRC +=\
 	$(SOURCEDIR)/Math/GPUWatcher.cu \
 	$(SOURCEDIR)/Math/GPURNGHandle.cu \
 	$(SOURCEDIR)/Math/MatrixQuantizerGPU.cu \
+	$(SOURCEDIR)/Math/NcclComm.cpp \
```

Comment:
To fix Windows build, please add the following into Source/Common/Common.vcxproj, line 65:
```
    <ClCompile Include="..\Math\NcclComm.cpp" />
```
(We'll need to figure out a better place in the end; but for now it seems the easiest change to fix Windows build)